src/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+  --active_star: url(images/star-active.svg);
+}
+
+.stars__block {
+  align-content: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: 96px;
+  width: 100px;
```

Comment:
You don't need this to do it flex and specify flex-wrap property, as the block takes 100%.